### PR TITLE
feat: ability to display image history in UI

### PR DESCRIPTION
**Pull request by rbreeze**
Stores on/off (per project) in localStorage

CleanShot 2023-09-21 at 11 55 50@2x

### CODE REVIEW

File: ui/src/utils/prefs.ts
Comment (rpelczar) [inline ui/src/utils/prefs.ts:1]: Can we name this file use-local-storage so we know it is a hook?

File: ui/src/features/project/project-details/images.tsx [hunk 1]
```diff
@@ -1,41 +1,138 @@
...
         }
-        stages.add(stage.metadata?.name as string);
+        stages[stage.metadata?.name as string] = {
+          opacity: 1,
+          backgroundColor: colors[stage.metadata?.uid as string]
+        };
       });
     });
-    return [images, getStageColors([...stages])];
+    return [images];
```
Comment (rpelczar) [inline ui/src/features/project/project-details/images.tsx:121]: This array is unnecessary here.

**Review by rpelczar — APPROVED** [review]
I added two minor comments.

File: ui/src/features/project/project-details/images.tsx (hunk 1 above)
Comment (rbreeze) [reply ui/src/features/project/project-details/images.tsx:121]: Thanks for catching that 🙂

**Pull request merged**